emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java

```diff
@@ -138,10 +136,9 @@
         }
       }
     });
-
     popupWindow.setContentView(emojiView);
+
     popupWindow.setInputMethodMode(PopupWindow.INPUT_METHOD_NOT_NEEDED);
-    popupWindow.setBackgroundDrawable(new BitmapDrawable(context.getResources(), (Bitmap) null)); // To avoid borders and overdraw.
```

Comment:
How come we can remove this?

Reply:
Because we do not pass the context to the PopupWindow afaik.